modules/cnv-template-vmi-config-with-masquerade.adoc

```diff
@@ -0,0 +1,46 @@
+// Module included in the following assemblies:
```

Comment:
This is an invalid spec, you can't create a VMI out of it.
For example - it doesn't contain name field.

Reply:
@yossisegev I've updated it. Is that better?

Reply:
@aburdenthehand trying to create a VM using this spec now fails on another error, at least in my env:
`Error from server (InternalError): error when creating "STDIN": Internal error occurred: failed calling webhook "mutatevirtualmachines.example.com": Post https://kubemacpool-service.kubemacpool-system.svc:443/mutate-virtualmachines?timeout=30s: stream error: stream ID 67; INTERNAL_ERROR`

Did you try deploying this spec on your env? If you did and it succeeded, then it's probably something in my env, and you can use this spec.
@SchSeba can you please take a look? It looks like something related to kubemacpool.
If you want to check it on my env - it's the same cluster which detais I sent you in my previous email (the one about masquerade). Thanks!

Reply:
@yossisegev
Right you are. I was able to test this in a borrowed environment. I've updated it again - this time taking the default config provided by the web console (Create via YAML), modified with the details in the previous example.
Using this config, I was able to create the VM and run it successfully.

Reply:
@aburdenthehand 
I have just tested the spec in modules/cnv-template-vm-config.adoc, and verified it creates a VM successfully.